hashintel/hash
# Bump the `@ory/client` version in the frontend

## 🌟 What is the purpose of this PR?
We've had a long-standing todo for bumping the ory client version to properly type the `code` verificaiton flow. This has recently been updated, so this PR is bumping the version and fixing some issues along the way that caused registrations not to work in the dev instance.

This PR shouldn't change any existing behavior, but fix some subtle issues we may have been running into.

## ❓ How to test this?



1.  Checkout the branch
1.  Start external services + `yarn dev` (follow the README for windows changes if running on WSL)
1.  Confirm that registration works as expected, go to http://localhost:4436/ and click the verification link
1.  Confirm that verification works as expected

## 📹 Demo



Verification still works
![image](https://github.com/hashintel/hash/assets/6988668/a9620392-f5c9-4adc-ae9d-6df324d9fcbf)

And code is now able to type check without assertion
